### PR TITLE
No attunement

### PR DESCRIPTION
**Pull request by PhilPJL**
Similar to 'no identify'.  

Apologies for tidying the code in the same commit.  Makes it hard to see what's changed.

### CODE REVIEW

**Review by ThyWoof — APPROVED** [review]

File: SolastaCommunityExpansion/Patches/GameUi/PowerSelectionPanelPatcher.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ internal static void Postfix(PowerSelectionPanel __instance)
                 {
                     if (thirdRow == null)
                     {
-                        thirdRow = GameObject.Instantiate(powersTable);
+                        thirdRow = Object.Instantiate(powersTable);
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Patches/GameUi/PowerSelectionPanelPatcher.cs:30]: Is switching from GameObject to Object safe here (and elsewhere)?

File: SolastaCommunityExpansion/Models/RemoveAttunementRequirementContext.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using SolastaModApi.Extensions;
+
+namespace SolastaCommunityExpansion.Models
+{
+    internal static class RemoveAttunementRequirementContext
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Models/RemoveAttunementRequirementContext.cs:5]: Should we have a context just for this (and some other one line changes) or merge them together? I could see this combined with the no identification change.

**Review by ChrisPJohn — CHANGES_REQUESTED** [review]
One question on the refactoring and one about how we want to organize going forward.

File: SolastaCommunityExpansion/Patches/GameUi/PowerSelectionPanelPatcher.cs (hunk 1 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Patches/GameUi/PowerSelectionPanelPatcher.cs:30]: Yes, I tend to trust VS refactorings.  Instantiate is implemented on Object not GameObject.

File: SolastaCommunityExpansion/Models/RemoveAttunementRequirementContext.cs (hunk 2 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Models/RemoveAttunementRequirementContext.cs]: Merged as suggested

**Review by ChrisPJohn — APPROVED** [review]
One question on the refactoring and one about how we want to organize going forward.

**Pull request merged**